src/QsCompiler/Transformations/MonomorphizationValidation.cs

```diff
@@ -53,14 +53,7 @@ private class NamespaceTransformation : NamespaceTransformation<TransformationSt
             public override QsCallable OnCallableDeclaration(QsCallable c)
             {
                 // Don't validate intrinsics
-                if (!SharedState.IntrinsicCallableSet.Contains(c.FullName))
-                {
-                    return base.OnCallableDeclaration(c);
-                }
-                else
-                {
-                    return c;
-                }
+                return SharedState.IntrinsicCallableSet.Contains(c.FullName) ? c : base.OnCallableDeclaration(c);
```

Comment:
nit: It would make it a bit easier for me to read if this were left as an if/else.

Reply:
Although, getting rid of the negation and swapping the statements works well.

Reply:
done